<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>restaurantownerorderitemwidget</class>
 <widget class="QWidget" name="restaurantownerorderitemwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 60, 60);
font: 12pt &quot;B Farnaz&quot;;
color: rgb(255, 85, 0);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>وضعیت سفارش</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>در حال آماده سازی</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>در حال ارسال</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="customerName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="foodText">
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string>  لذزبذبلذ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="restaurantName">
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string>  بیبلاتن</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
